mlir/include/mlir/Target/LLVMIR/ModuleImport.h

```diff
@@ -474,6 +481,10 @@ class ModuleImport {
   /// emitted. Avoids generating warnings for unhandled debug intrinsics and
   /// metadata that otherwise dominate the translation time for large inputs.
   bool emitExpensiveWarnings;
+
+  /// An option to control whether to disable supported intrinsic support in
+  /// favor a more generic version via `llvm.intrinsic_call`.
```

Comment:
```suggestion
  /// An option to control whether the importer should try to convert all intrinsics to
  /// llvm.call_intrinsic instead of dialect supported operations.
```
nit I prefer the comment from the function.